node/src/actors/chain_manager/mod.rs

```diff
@@ -615,6 +615,9 @@ fn update_pools(
         }
     }
 
+    // Remove commits because they expire every epoch
+    transactions_pool.clear_commits();
```

Comment:
This line is only running during block consolidation, so if there are not blocks in a epoch, it could be produce that there are commits from different epochs